test/views/ttfb.njk

```diff
@@ -27,13 +27,54 @@
   <script>
     // Set the blocking values based on query params if present.
     const params = new URLSearchParams(location.search);
+    const navEntry = performance.getEntriesByType('navigation')[0];
 
     if (params.has('responseStart')) {
-      const navEntry = performance.getEntriesByType('navigation')[0];
       Object.defineProperty(navEntry, 'responseStart', {
         value: Number(params.get('responseStart')),
+        enumerable: true,
+        writeable: true,
       });
     }
+
+    function block(blockingTime) {
+      const startTime = performance.now();
+      while (performance.now() < startTime + blockingTime) {
+        // Block...
+      }
+    }
+
+    if (params.has('earlyHintsDelay')) {
```

Comment:
Instead of stubbing this on the client, why not implement it on the server? Node.js [supports early hints](https://nodejs.org/api/http.html#responsewriteearlyhintshints-callback), and I was able to get it to work pretty easily by using the `response.writeEarlyHints()` method in the `server.js` file (which uses Express, but Express is just sugar on top of Node's HTTP server implementation, so every Express `Request` and `Response` objects are actually node `Request` and `Response` objects).

Reply:
Oh that was a lot easier! Thanks.